pasha_modules/util.coffee

```diff
@@ -218,11 +218,11 @@ pagerdutyAlert = (description) ->
 
 
 startNag = (adapter, msg) ->
-    state = getOrInitState(adapter)
-    prio1 = state.prio1
     naggerCallbackId = null
     nagger = () ->
-        if (not getOrInitState(adapter).prio1?)
+        state = getOrInitState(adapter)
```

Comment:
The state can and will change between when the `startNag` and `nagger` are called; for example, `state.prio1.channel` is set between those times.